csharp/ql/test/library-tests/csharp9/typeParameterNullability.ql

```diff
@@ -0,0 +1,27 @@
+import csharp
+
+query predicate refType(TypeParameter tp) {
+  tp.fromSource() and
+  (
+    not exists(tp.getConstraints())
```

Comment:
Should this be removed from all the test predicates? It seems that all type parameters in your test have constraints, and if there were one without, it would end up in all test predicates.

Reply:
Thanks.